src/components/docs/OptionsTable.tsx

```diff
@@ -11,7 +11,7 @@ export const OptionsTable: FC<{ children: ReactNode }> = ({ children }) => {
 
 export const OptionTitle: FC<{ children: ReactNode }> = ({ children }) => {
   return (
-    <div className="option-title hyphens not-prose -mt-px border-t border-gray-200 bg-gray-50 p-4 pt-5 dark:border-gray-800 dark:bg-gray-900/75 md:border-r lg:border-r-0 xl:border-r">
+    <div className="option-title hyphens not-prose -mt-px border-t border-gray-200 bg-gray-50 p-4 pt-5 dark:border-gray-800 dark:bg-gray-900/75 md:border-r lg:border-r-0 xl:border-r md:col-span-2">
```

Comment:
We have particular long strings in the first column for this page (e.g. `espressoAndCoffeeBottomText`). Quick fix was to set Col A and Col B to 50%. 

**Before:**
<img width="921" alt="image" src="https://user-images.githubusercontent.com/1712435/181696454-99f1d49c-d5da-4eb9-961f-0ae446954943.png">


**After:**
![image](https://user-images.githubusercontent.com/1712435/181696470-425a5cb0-b212-4a74-ab8f-ffca28eee9ce.png)

Reply:
Could there be a way to set the col width per table in the future?

Reply:
That would be cool 👍 

Card here: https://trello.com/c/HklJ7JLP/4547-dev-docs-add-col-span-prop-or-similar-to-table-component